src/adhocracy_core/docs/deletion.rst

```diff
@@ -1,123 +1,123 @@
-version as obsolete. POSTing a resource with *IMetadata { hidden: true }*
-requires the *hide_resource* permission and, frankly, doesn't make any sense.
-
 The effect of these flags is as follows:
 
-* A positive value of the *deleted* and *hidden* flags is inherited by
-  child/descendant resources (whose resource path includes the path of the
-  deleted/hidden ancestor as prefix). Hence a *deleted* resource is one
-  that has its own *deleted* flag set to true or that has an ancestor whose
-  *deleted* flag is true. Likewise for *hidden* resources.
-* Normally, only resources that are neither *deleted* nor *hidden* are
-  listed in parent pools and search queries.
-* FIXME Not implemented yet, since the frontend doesn't yet need it:
-  The parameter *include=deleted|hidden|all* can be used to include
-  deleted and/or hidden resources in pool listings and other search queries.
-  If its value is *deleted*, resources will be found regardless of the value
-  of their *deleted* flag, but only if they are not *hidden.* If its value is
-  *hidden*, resources will be found regardless of the value of their *hidden*
-  flag, but only if they are not *deleted.* If its value is *all*, all
-  resources will be found. Anyone can specify these flags and get the paths
-  of deleted and/or hidden resources. However, only those with *hide_resource*
-  permission are ever able to view the contents of hidden resources.
-  It's also possible to set *include=visible* to get only non-deleted and
-  non-hidden resources, but it's not necessary since that is the default.
-* If the frontend attempts to retrieve a *deleted* or *hidden* resource via
-  GET, the backend normally responds with HTTP status code *410 Gone*.
-  FIXME Not implemented yet, since the frontend doesn't yet need it:
-  The frontend can override this by adding the parameter
-  *include=deleted|hidden|all* to the GET request, just as in search queries.
-  Anybody can view deleted resources in this way, and managers (those with
-  *hide_resource* permission) can view hidden resources in these ways. Those
-  without this permission will still get a *410 Gone* if the resource is
-  hidden.
-* The body of the *410 Gone* is a small JSON document that explains why the
-  resource is gone (whether it was deleted or hidden). It also shows who
-  made the last change to the resource and when::
-
-      { "reason": "deleted|hidden|both",
+* A positive value of the *hidden* flags is inherited by
+  child/descendant resources (whose resource path includes the path of
+  the hidden ancestor as prefix). Hence a *hidden* resource is one that
+  has its own *hidden* flag set to true or that has an ancestor whose
+  *hidden* flag is true.
+* Normally, only resources that are not *hidden* are listed in parent
+  pools and search queries.
+* FIXME Not implemented yet, since the frontend doesn't yet need it: The
+  parameter *include=hidden* can be used to include hidden resources in
```

Comment:
this paragraph also mentions the old "delete" . 
We can remove this paragraph, we wont fix this fixme anyway not soon.